config/environments/production.rb

```diff
@@ -50,6 +62,12 @@
   # Use a different cache store in production.
   config.cache_store = :redis_cache_store, REDIS_CACHE_PARAMS
 
+  # Use a real queuing backend for Active Job (and separate queues per environment).
+  # config.active_job.queue_adapter     = :resque
+  # config.active_job.queue_name_prefix = "mastodon_production"
+
+  config.action_mailer.perform_caching = false
```

Comment:
Enables fragment caching in mailer templates https://edgeguides.rubyonrails.org/configuring.html#config-action-mailer-perform-caching